css/properties/appearance.json

```diff
@@ -506,10 +516,15 @@
                 "version_added": "1",
                 "partial_implementation": true
               },
-              "samsunginternet_android": {
-                "version_added": "1.0",
-                "partial_implementation": true
-              },
+              "samsunginternet_android": [
+                {
+                  "version_added": "13.2"
```

Comment:
```suggestion
                  "version_added": "13.0"
```

Reply:
Good catch I just assumed 13.0 was on an Earlier chromium version. Corrected that now